grc/gui/PropsDialog.py

```diff
@@ -215,25 +215,10 @@ def _update_docs_page(self):
         if not docstrings:
             return
 
-        # show documentation string from block xml
-        from_xml = docstrings.pop('', '')
-        for line in from_xml.splitlines():
-            if line.lstrip() == line and line.endswith(':'):
```

Comment:
This did some special formatting for the docstrings extracted from the grc block definition. (Lines ending with `:`) are printed in bold.